android/KMAPro/kMAPro/src/main/java/com/tavultesoft/kmapro/KMPBrowserActivity.java

```diff
@@ -162,6 +163,15 @@ protected void onPause() {
   @Override
   protected void onDestroy() {
     super.onDestroy();
+    // Blank the webView and destroy completely
```

Comment:
Did you snarf this pattern from somewhere online? If so, good to reference that.

It looks similar to https://stackoverflow.com/a/17458577/1836776

Reply:
Documented in WebViewUtil.java